roles/engineer.md

```diff
@@ -12,37 +12,14 @@ We need someone:
 
 We expect someone in the role:
 
+- to be billable for at least half their time after 2 weeks on-boarding
```

Comment:
Yeah I changed this here to provoke this conversation actually. I'm not sure how to document this difference in expectation.